src/components/Accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import {LayoutAnimation, View, TouchableOpacity, Text} from 'react-native'
+import {toggleAnimation} from './ToggleAnimation'
+import styled from 'styled-components/native'
+import type {ITheme} from '../../theme'
+import type {section, CommonAccordionProps} from './Accordion'
+
+export type Theme = {
+  theme?: ITheme
+}
+
+export interface AccordionItemProps extends CommonAccordionProps {
+  /**
+   * content of accordion
+   */
+  children?: React.ReactElement
+  /**
+   * onPress item event
+   */
+  onPress: (keyExtractorItem: string) => void
+  /**
+   * keyExtractor for item
+   */
+  keyExtractorItem: string
+  expanded?: boolean
+  index: number
+  item: section
+}
+
+const AccordionItem = ({
+  children,
+  title = '',
+  onPress,
+  keyExtractorItem,
+  expanded = false,
+  index,
+  itemContainerStyle,
+  headerContainerStyle,
+  contentContainerStyle,
+  titleStyle,
+  item,
+  openAnimation,
+  closeAnimation,
+  openDuration,
+  closeDuration,
+  renderHeader = () => null,
+  renderSectionTitle,
+  renderContent,
+}: AccordionItemProps) => {
+  let content = null
+
+  if (expanded) {
+    if (renderContent) {
+      content = renderContent(item, index, expanded, keyExtractorItem)
+    } else {
+      content = <AccordionBody style={contentContainerStyle}>{children}</AccordionBody>
+    }
+  }
+
+  const header = renderHeader(item, index, expanded, keyExtractorItem) || (
+    <AccordionHeader style={headerContainerStyle}>
+      {renderSectionTitle ? (
+        renderSectionTitle(item, index, expanded)
+      ) : (
+        <Title style={titleStyle}>{title}</Title>
+      )}
+    </AccordionHeader>
+  )
+
+  return (
+    <AccordionContainer style={itemContainerStyle}>
+      <TouchableOpacity
+        testID="accordion-item"
+        activeOpacity={0.7}
```

Comment:
create constant for this value

Reply:
why we need to create constant for activeOpacity value ?. It's not magic number and we do not use this value multiple time